src/cmap/connect.ts

```diff
@@ -129,6 +131,25 @@ function performInitialHandshake(
         return;
       }
 
+      if (options.loadBalanced) {
+        // TODO: Durran: Remove when server support exists. (NODE-3431)
+        if (FAKE_MONGODB_SERVICE_ID) {
+          if (response.topologyVersion) {
```

Comment:
this might look prettier as a ternary:
`response.serviceId = response.topologyVersion ? response.topologyVersion.processId : new ObjectId();`

for my own understanding, could you explain why we fall back to `new ObjectId()` specifically if `topologyVersion` is not returned?

Reply:
This is now updated to always use `processId`.